src/Dapr.AspNetCore/CloudEventsMiddleware.cs

```diff
@@ -27,6 +30,14 @@ namespace Dapr
     internal class CloudEventsMiddleware
     {
         private const string ContentType = "application/cloudevents+json";
+
+        // These cloudevent properties are either containing the body of the message or 
+        // are included in the headers by other components of Dapr earlier in the pipeline 
+        private static readonly string[] ExcludedPropertiesFromHeaders =
```

Comment:
Might there be other properties that users want excluded for size/performance reasons?  Would it make sense for the options object to offer a property to set additional exclusions and/or included properties?

Reply:
that is an interesting idea, ill think about it for a bit
maybe two complementary optional lists with included and excluded property names will make it a lot more flexible
for cases where users might want to either forward a,b,c props only, or forward everything except a,b,c